gcc.cmake

```diff
@@ -26,10 +26,34 @@ set(sel4_arch @KernelSel4Arch@)
 set(arch @KernelArch@)
 set(mode @KernelWordSize@)
 
+# This function hunts for an extant `gcc` with one of the candidate prefixes
+# specified in `ARGN`, allowing us to try different target triple prefixes for
+# cross-compilers built in various ways.
+function(FindPrefixedGCC out_var)
```

Comment:
If it's better to put this helper somewhere else I'm happy to move it.

Reply:
I'm fine with this location